package.json

```diff
@@ -61,7 +63,8 @@
     "test": "npm-run-all dist js-test docs-compile docs-lint bundlesize",
     "watch": "npm-run-all --parallel watch-css watch-js",
     "watch-css": "nodemon --ignore js/ --ignore dist/ -e scss -x \"npm run css\"",
-    "watch-js": "nodemon --ignore scss/ --ignore js/dist/ --ignore dist/ -e js -x \"npm run js-compile\""
+    "watch-js": "nodemon --ignore scss/ --ignore js/dist/ --ignore dist/ -e js -x \"npm run js-compile\"",
+    "server": "http-server -s -p 3000"
```

Comment:
Not sure this makes sense in this branch since we already need Ruby.

I did this in my Hugo branch because the Ruby dep is removed.

Reply:
OK, now I read you comment about phantom.

Why do we need phantom anyway? Can't we use jsdom or Chrome/Firefox headless?

Reply:
Yep that's a possible switch because `phantomjs` won't receive any fixes